src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -735,6 +735,23 @@ public static function with($relations)
 
         return $instance->newQuery()->with($relations);
     }
+    
+    /**
+     * Append attributes to query when building a query.
+     * 
+     * @param  array|string  $relations
+     * @return $this
+     */
+    public function append($relations)
```

Comment:
Calling the parameter `$relations` seems wrong to me though. Those could be dynamic attributes (via accessor) as well... How about `$attributes`?